system/Test/FeatureTestCase.php

```diff
@@ -57,7 +57,7 @@ protected function withRoutes(?array $routes = null)
     {
         $collection = Services::routes();
 
-        if ($routes) {
+        if ($routes !== null && $routes !== []) {
```

Comment:
cast to array

Reply:
Only `$routes !== null` is sufficient.

Reply:
@kenjis it based on type declaration

```php
    protected function withRoutes(?array $routes = null)
    {
```

Reply:
There are two options: specify routes or do not specify routes.
`null`: do not specify
`array`: specify routes

We do not need to accept `[]` here. If you don't specify routes, pass `null`.

Reply:
updated to ony check !==null